community/kernel/src/test/java/org/neo4j/kernel/impl/index/schema/NativeSchemaIndexPopulatorTest.java

```diff
@@ -599,7 +598,8 @@ private void verifyUpdates( Iterator<IndexEntryUpdate<SchemaIndexDescriptor>> in
     private byte[] fileWithContent() throws IOException
     {
         int size = 1000;
-        try ( StoreChannel storeChannel = fs.create( indexFile ) )
+        fs.mkdirs( getIndexFile().getParentFile() );
```

Comment:
The structure of the indexes somehow got deeper (with a folder for the provider (e.g. lucene-1.0) and a subfolder for the sub-provider (e.g. temporal-1.0). Creating the file of these deeper nested directories requires creating the directories first. This was not true for Number/string/lucene, since they don't have sub-providers.